controller/src/main/java/org/jboss/as/controller/audit/ManagedAuditLogger.java

```diff
@@ -182,6 +184,14 @@
     JsonAuditLogItemFormatter getJsonFormatter(String name);
 
     /**
+     * Gets the last log entries
+     *
+     * @param name the name of the handler
+     * @return the last log entries of the handler
+     */
+    List<ModelNode> listLastEntries(String name);
```

Comment:
Cool it's not a huge deal, but just better for compatibility. Even though in this case it won't really matter.